vite.config.js

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import path from 'path';
+
+export default defineConfig({
+    plugins: [react()],
+    resolve: {
+        alias: {
+            src: path.resolve(__dirname, 'src'),
```

Comment:
That is a great point for absolute imports. In general, we use imports related to the src folder, like import Button' from 'component/button

Reply:
added path aliases